MapboxGeocoder/MBGeocodeOptions.swift

```diff
@@ -46,6 +46,11 @@ public class GeocodeOptions: NSObject {
      */
     public var allowedRegion: RectangularRegion?
     
+    /**
+     Limit the number of results returned. The default is `5` for forward geocoding and `1` for reverse geocoding.
+     */
+    public var maximumResultCount: UInt = 0
```

Comment:
Turns out this _is_ required if we want to keep the superclass initializer implicit: 

https://www.bitrise.io/build/891808d9622c3599